partiql-test-data/pass/parser/primitives/call.ion

```diff
@@ -50,42 +50,46 @@ parse::{
 
 
 'substring'::[
-    parse::{
-        name: "call SUBSTRING sql92 syntax",
-        statement: "substring('test' from 100)",
-        assert: [
-            {
-                result: ParseOk
-            },
-        ]
-    },
-    parse::{
-        name: "call SUBSTRING sql92 syntax with length",
-        statement: "substring('test' from 100 for 50)",
-        assert: [
-            {
-                result: ParseOk
-            },
-        ]
-    },
-    parse::{
-        name: "call SUBSTRING normal syntax",
-        statement: "substring('test', 100)",
-        assert: [
-            {
-                result: ParseOk
-            },
-        ]
-    },
-    parse::{
-        name: "call SUBSTRING normal syntax with length",
-        statement: "substring('test', 100, 50)",
-        assert: [
-            {
-                result: ParseOk
-            },
-        ]
-    },
+    'sql92'::[
+        parse::{
+            name: "call SUBSTRING sql92 syntax",
+            statement: "substring('test' from 100)",
+            assert: [
+                {
+                    result: ParseOk
+                },
+            ]
+        },
+        parse::{
+            name: "call SUBSTRING sql92 syntax with length",
+            statement: "substring('test' from 100 for 50)",
+            assert: [
+                {
+                    result: ParseOk
+                },
+            ]
+        },
+    ],
+    'function-argument-list'::[
```

Comment:
Thanks for breaking these down, since we'll likely going to have a similar pattern in other tests (E.g. `sql92` vs. `sql2011` vs. partiql), does it make sense to rename such namespaces (E.g. `function-argument-list`) to _partiql_?

Reply:
Ok, sure I can change it to the `partiql` namespace.